File: totosuki/ABC/B/237.py
import sys
input = sys.stdin.buffer.readline

H, W = map(int, input().split())
A = [list(map(int, input().split())) for _ in range(H)]
B = [[0] * H for _ in range(W)]

for i in range(H):
  for j in range(W):
    B[j][i] = A[i][j]

[print(*b) for b in B]

File: OCH_Tao/ABC/B/237.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [, W] = input.shift().split(" ").map(x => parseInt(x, 10));
let A = [];
for (const i of input) {
  A.push(i.split(" "));
}
for (let i = 0; i < W; i++) {
  console.log(A.map(x => x[i]).join(" "));
}